packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerFooter.tsx

```diff
@@ -78,62 +78,81 @@ export const TimePickerFooter = (props: Props) => {
           variant="secondary"
           onClick={onToggleChangeTimeSettings}
           size="sm"
+          aria-expanded={isEditing}
+          aria-controls="time-settings-section"
```

Comment:
also just thought, couldn't we just use `useId` here? e.g.

```tsx
const timeSettingsId = useId();

// ...

<Button
  aria-controls={timeSettingsId}
>

// ...

<div className={style.editContainer} id={timeSettingsId}>
```

Reply:
I've run into som issues with useId, where pa11y fails because it doesn't think `:` should be in a valid id. I guess we could write a wrapper around it.

Reply:
ah i see, basically hitting https://github.com/facebook/react/issues/26839

good to know 👍